docs/root/version_history/current.rst

```diff
@@ -37,7 +37,14 @@ Removed Config or Runtime
 New Features
 ------------
 
+* bootstrap: added :ref:`dns_resolver_options <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dns_resolver_options>` to aggregate all of the DNS resolver options in a single message. By setting one such option *no_default_search_domain* as true the DNS resolver will not use the default search domains.
+* cluster: added :ref:`dns_resolver_options <envoy_v3_api_field_config.cluster.v3.Cluster.dns_resolver_options>` to aggregate all of the DNS resolver options in a single message. By setting one such option *no_default_search_domain* as true the DNS resolver will not use the default search domains.
+* dynamic_forward_proxy: added :ref:`dns_resolver_options <envoy_v3_api_field_extensions.common.dynamic_forward_proxy.v3.DnsCacheConfig.dns_resolver_options>` to aggregate all of the DNS resolver options in a single message. By setting one such option *no_default_search_domain* as true the DNS resolver will use the default search domains.
```

Comment:
Why are we adding the new options message to these additional places?  Wouldn't the cluster and forward proxy reference the config in the bootstrap?

Reply:
Even though the protobuf message is present in the base.proto, the deprecated field _use_tcp_for_dns_lookups_ is present in all 3 proto files and the new field _dns_resolver_options_ is is added in all 3 proto files (dns_cache.proto, bootstrap.proto, cluster.proto). I just wanted to call out that these API changes are done on all three places.

What would be better approach? Since these three statements are just repetition should I make it a single point and mention the proto locations where they are used?